cartopy/documentation.html

```diff
@@ -48,19 +49,34 @@
           "Getting started" and some step-by-step examples.</p>
           <h2>Latest</h2>
             <ul>
-                <li><a href="docs/latest/index.html">v0.14</a></li>
+                <li><a href="docs/latest/index.html">v0.13</a></li>
```

Comment:
This evidently isn't correct! I'm going to blame this on the version of Cartopy that was active in the Python environment that I ran the build script with, change that version, and push a commit that fixes this.